lib/config/screen/pre_fare.ex

```diff
@@ -39,7 +43,9 @@ defmodule ScreensConfig.Screen.PreFare do
             evergreen_content: [],
             content_summary: nil,
             cr_departures: CRDepartures.from_json(:default),
-            shuttle_bus_info: ShuttleBusInfo.from_json(:default)
+            shuttle_bus_info: ShuttleBusInfo.from_json(:default),
+            enable_departures: false,
+            departures: nil
```

Comment:
Since it appears we are already allowing `departures` to be `nil` (contra the typespec though), can we use this as our indicator of whether departures are enabled, rather than having a separate boolean flag? Then there is no way for the config to be ambiguous/contradictory (e.g. departures is `nil` but `enable_departures` is true).